milli/src/index.rs

```diff
@@ -203,6 +203,25 @@ impl Index {
         Ok(self.main.get::<_, Str, SerdeJson<FieldsIdsMap>>(rtxn, FIELDS_IDS_MAP_KEY)?.unwrap_or_default())
     }
 
+    /* fields ids distribution */
+
+    /// Returns the fields ids distribution which associate the internal field ids
+    /// with the number of times it occurs in the obkv documents.
+    // TODO store in the index itself and change only within updates that modify the documents
+    pub fn fields_ids_distribution(&self, rtxn: &RoTxn) -> anyhow::Result<HashMap<FieldId, usize>> {
```

Comment:
```suggestion
    pub fn fields_ids_distribution(&self, rtxn: &RoTxn) -> anyhow::Result<HashMap<FieldId, u64>> {
```

It could be maybe strange but as the documents and document fields are stored on disk, there can be more that 2^32 field occurrences, therefore using an `usize` on a 32bit machine could be a problem, using an u64 is safe.

Reply:
Got it, completely agree. Fixed.